site/README.md

```diff
@@ -74,3 +74,8 @@ PATH_PREFIX='/hds-demo/docsite-fixes' yarn build -- --prefix-paths
 ## Troubleshooting
 - **Site build fails with error #11903 COMPILATION or something similar**. Gatsby's cache might not be in sync with the source files due to a checkout (chancing branches, etc.)  
   **Solution:** Run `$(npm bin)/gatsby clean` or manually remove .cache folder to clean up cache files. Then try to run the build again.
+
+- **Development: A new local component is not available when trying to import from the hds-react in the site**:
+  1. Review that all component exports exist in hds-react package
```

Comment:
Should we mention specifically the component and components folder's index.ts files so there's no ambiguity?

Reply:
How about adding a script for this to the <root>/package.json

`"rebuild-site": "yarn --cwd site clean && yarn build && yarn --cwd site start"`

I guess `lerna run` could also be used, but I'm not familiar with that.

Reply:
Also requirement to add new css to all.css is not documented.

Reply:
I will add a link to development.md

Reply:
The exports are explained in here: https://github.com/City-of-Helsinki/helsinki-design-system/blob/8f9342a8a2a5a1fdaad35025fa2e82cf5c698065/packages/react/DEVELOPMENT.md